specificatie/openapi.yaml

```diff
@@ -648,6 +828,94 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Zoekobject om te zoeken op geometrie.'
+      type: object
+      properties:      
+        geometrie:
+          $ref: 'http://schemas.opengis.net/ogcapi/features/part1/1.0/openapi/schemas/polygonGeoJSON.yaml'
```

Comment:
@CathyDingemanse @MelvLee is er ook behoefte aan een punt met straal (weet niet of dit in geoJson bestaat)? Of vinden we dat een gebruiker/developer dit zelf kan omzetten naar polygoon?

Reply:
@strijm is dit ook mogelijk, om zoeken op punt met straal toe te voegen? Dan kan een gebruiker onderweg de dichtstbijzijnde panden/verblijfsobjecten zien waar deze naartoe kan. @CathyDingemanse geeft aan dat dit wel wenselijk is.

Reply:
Is toegevoegd.